addons/mail/static/src/discuss/discuss.js

```diff
@@ -16,8 +17,17 @@ export class Discuss extends Component {
 
     setup() {
         this.messaging = useMessaging();
+        this.state = useState({ threadRename: "", isHoveringThreadName: false });
+        this.threadNameInputRef = useRef("threadNameInput");
+        useExternalListener(document, 'click', this._onClickCaptureGlobal, { capture: true })
```

Comment:
Note: should use prettier. Here it would use double quotes rather than single quote.

Reply:
do we have a config file for the prettier?